tuf/utils/x509.go

```diff
@@ -443,6 +454,9 @@ func EncryptPrivateKey(key data.PrivateKey, role, passphrase string) ([]byte, er
 		return nil, fmt.Errorf("unable to encrypt key - invalid PEM file produced")
 	}
 	encryptedPEMBlock.Headers["role"] = role
+	if gun != "" {
```

Comment:
@cyli's right that we required a role when importing a key if one did not exist in the PEM headers, so in our docs we suggested `"user"`, but it wasn't hardcoded anywhere.

I'd like to make sure that the new `notary key import` can import delegation keys directly from openssl so maybe we can have a default role value or no header at all?